lib/find.js

```diff
@@ -102,7 +102,13 @@ function _find(tree, _options) {
   var options, root;
 
   if (arguments.length === 1) {
-    var decomposition = decompose(tree);
+    var decomposition;
+
+    if (typeof tree === 'object') {
+      decomposition = decompose(tree.inputTree);
```

Comment:
How I feel a lot of times with Broccoli:
![](http://i.kinja-img.com/gawker-media/image/upload/s--qOw8koA5--/japbcvpavbzau9dbuaxf.jpg)

Anyways I think what we need to do is check if the first argument is an object, if is we just use it as the root for the funnel passing the decompose options.